snuba/cli/devserver.py

```diff
@@ -55,6 +55,18 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 "--consumer-group=sessions_group",
             ],
         ),
+        (
+            "outcomes-consumer",
+            [
+                "snuba",
+                "consumer",
+                "--auto-offset-reset=latest",
+                "--log-level=debug",
+                "--storage=outcomes_raw",
+                "--consumer-group=outcomes_group",
+                "--commit-log-topic=snuba-commit-log",
```

Comment:
The outcome consumer does not use the commit log. Could you please remove this line before merging ?